guacamole/src/main/webapp/app/navigation/directives/guacUserMenu.js

```diff
@@ -160,9 +219,9 @@ angular.module('navigation').directive('guacUserMenu', [function guacUserMenu()
             };
 
             /**
-             * All available actions for the current user.
+             * Add logout action for the current user.
              */
-            $scope.actions = [ LOGOUT_ACTION ];
+            $scope.actions = $scope.actions.concat( LOGOUT_ACTION );
```

Comment:
Rather than repeatedly recreating the `$scope.actions` array through calling `concat()`, perhaps it would be better to keep the original initialization to `[ LOGOUT_ACTION ]` and simply `unshift(FULLSCREEN_ACTION)` when needed?